INFO - 2021-09-15 10:27:26,536 - new_apply_order.py[line:646]: 开始第 pcamion【1】次进件……
INFO - 2021-09-15 10:27:26,536 - new_apply_order.py[line:82]: ======【co测试环境】生成手机号：3631672846 ======
INFO - 2021-09-15 10:27:26,942 - new_apply_order.py[line:95]: 3631672846 检测用户存在成功！
ERROR - 2021-09-15 10:27:30,119 - new_apply_order.py[line:106]: {"msg":"The Server is busy, Please try again later","code":"9999","data":null} 短信发送失败！
ERROR - 2021-09-15 10:27:30,161 - new_apply_order.py[line:119]: 注册失败！{'accept': 'application/json', 'appversion': '1.3.0.6', 'devtoken': 'cb95881f542ac58170:BB:E9:91:D3:CD', 'accept-language': 'en-US', 'clientCode': 'moremoney1300', 'Content-Type': 'application/json; charset=UTF-8', 'Host': '172.20.240.245:8012', 'User-Agent': 'okhttp/4.9.0'}
ERROR - 2021-09-15 10:27:30,161 - new_apply_order.py[line:120]: {"code":"60001004","msg":"The OTP has expired.please regenerate OTP to try again","data":null}
ERROR - 2021-09-15 10:27:33,211 - new_apply_order.py[line:135]: {"msg":"The Server is busy, Please try again later","code":"9999","data":null} 登录短信发送失败！
ERROR - 2021-09-15 10:27:33,248 - new_apply_order.py[line:150]: 登录失败!
ERROR - 2021-09-15 10:27:33,251 - new_apply_order.py[line:692]: 'data_fillInUserInfo_mx_co'
ERROR - 2021-09-15 10:27:33,253 - new_apply_order.py[line:693]: None
